apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -0,0 +1,298 @@
+                  Subscriber profile Image can only be updated via API
+                </TooltipContent>
+              </Tooltip>
+              <div className="flex flex-1 items-center gap-2.5">
+                <FormField
+                  control={form.control}
+                  name="firstName"
+                  render={({ field, fieldState }) => (
+                    <FormItem className="w-full">
+                      <FormLabel>First Name</FormLabel>
+                      <FormControl>
+                        <Input
+                          {...field}
+                          placeholder={field.name}
+                          id={field.name}
+                          value={field.value}
+                          onChange={field.onChange}
+                          hasError={!!fieldState.error}
+                        />
+                      </FormControl>
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name="lastName"
+                  render={({ field, fieldState }) => (
+                    <FormItem className="w-full">
+                      <FormLabel>Last Name</FormLabel>
+                      <FormControl>
+                        <Input
+                          {...field}
+                          placeholder={field.name}
+                          id={field.name}
+                          value={field.value}
+                          onChange={field.onChange}
+                          hasError={!!fieldState.error}
+                        />
+                      </FormControl>
+                    </FormItem>
+                  )}
+                />
+              </div>
+            </div>
+            <div>
+              <FormItem className="w-full">
+                <div className="flex items-center">
+                  <FormLabel
+                    tooltip="Provide a unique ID for the user as the subscriberId (e.g., your app's internal user ID)."
+                    className="gap-1"
+                  >
+                    SubscriberId
+                  </FormLabel>
+                  <span className="ml-auto">
+                    <Link
+                      to="https://docs.novu.co/concepts/subscribers"
+                      className="text-xs font-medium text-neutral-600 hover:underline"
+                      target="_blank"
+                    >
+                      How it works?
+                    </Link>
+                  </span>
+                </div>
+                <Input value={subscriberId} readOnly trailingNode={<CopyButton valueToCopy={subscriberId} />} />
+              </FormItem>
+            </div>
+
+            <div className="flex flex-1 items-center gap-2.5">
+              <FormField
+                control={form.control}
+                name="email"
+                render={({ field, fieldState }) => (
```

Comment:
no error is shown in the case of an invalid email
![Screenshot 2025-02-04 at 09 42 50](https://github.com/user-attachments/assets/6804b313-9a13-48bd-b1df-2fdb2b7e5537)